CHANGELOG.md

```diff
@@ -172,6 +201,7 @@
 * Fix/Improve snapping to beats in various situations [#4366](https://github.com/mixxxdj/mixxx/pull/4366) [lp:1945238](https://bugs.launchpad.net/mixxx/+bug/1945238)
 * Allow to cancel active loops via beatloop_activate [#4328](https://github.com/mixxxdj/mixxx/pull/4328) [lp:1876003](https://bugs.launchpad.net/mixxx/+bug/1876003)
 * Don't wipe inapplicable sound config immediately [#4544](https://github.com/mixxxdj/mixxx/pull/4544)
+* Allow to reset loops via "[ChannelN], loop_remove" control object [#4802](https://github.com/mixxxdj/mixxx/pull/4802) [lp:1978507](https://bugs.launchpad.net/mixxx/+bug/1978507)
```

Comment:
This should go to below the other **Track Menu** entries, though just to improve discoverability?
Or we add it twice: the track menu action to **Library** and the control here?